manifests/cf-manifest/spec/fixtures/cf-dev.yml

```diff
@@ -0,0 +1,6 @@
+---
+meta:
+  cell_z1:
+    instances: 1
```

Comment:
From meatspace conversation with @mtekel: we're going to move the attributes for dev/CI/staging to a "defaults" file, which will reduce duplication and simplify the possible combinations of tests that we need to write in the future.